tests/bofire/test_benchmark.py

```diff
@@ -0,0 +1,44 @@
+from functools import partial
+
+import pandas as pd
+
+import bofire.benchmarks.benchmark as benchmark
+from bofire.benchmarks.zdt import ZDT1
+from bofire.samplers import RejectionSampler
+from bofire.strategies.botorch.sobo import BoTorchSoboStrategy, qEI
+
+
+def test_benchmark():
+    zdt1 = ZDT1(n_inputs=5)
+    sobo_factory = partial(BoTorchSoboStrategy, acquisition_function=qEI())
```

Comment:
Does it make sense to use here sobo for an MO objective?

Reply:
> Does it make sense to use here sobo for an MO objective?

I thought Sobo just multiplies the objectives in this case?

Reply:
Yes, it does, but does this make sense for an explicipt MO benchmark function? I would recommend to use QEHVI or qPAREGO.

Reply:
Ok. I can change to qPAREGO or the like. This was just meant to be a technical test.

Reply:
Perfect!